rts/lra/proxy/api/pom.xml

```diff
@@ -59,7 +59,6 @@
     <dependency>
       <groupId>org.jboss.resteasy</groupId>
       <artifactId>resteasy-client</artifactId>
-      <scope>test</scope>
```

Comment:
I needed to remove the scope 'test' otherwise the compilation fails after removing scope from dependencies' bom.